src/table/utils/accessibility-utils.ts

```diff
@@ -216,9 +216,14 @@ export const announceSelectionState = (
   }
 };
 
-export const copyCellValue = async (value: string) => {
+export const copyCellValue = async (evt: any, isHeadCell: boolean = false) => {
+  const value =
+    isHeadCell && (evt.target as HTMLElement).children[0].firstChild
```

Comment:
It do seem a bit clunky and hard ta read when you do the type casting 4 times for the same property. Why not just store the casted value in a variable? Ex: `const target = evt.target as HTMLElement`. If linting complains I'd probably just disable it for this line (not sure if you can to type casting when you do object destructing..)